Automattic/vip-go-mu-plugins
# Skip confirm

## Description


## Changelog Description

### Plugin Updated: Search

Adds a `--skip-confirm` for `wp vip-search index-versions activate ..` command. To be able to run the command programatically.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1. Create version `wp vip-search index-versions add post`
1. Activate the new version `wp vip-search index-versions activate post 2` - shows confirmation
1. Activate the new version `wp vip-search index-versions activate post 2 --skip-confirm` - Does not show confirmation
